gr-vocoder/lib/gsm_fr_decode_ps_impl.h

```diff
@@ -26,7 +26,7 @@
 #include <gnuradio/vocoder/gsm_fr_decode_ps.h>
 
 extern "C"{
-#include "gsm/gsm.h"
+#include "gsm.h"
```

Comment:
Not yet convinced this non-standard includery is necessary.

Reply:
agreeing with @mboersch here.